### ISSUE TITLE
NoMethodError: private method include called for ActionDispatch::Request:Class

### ISSUE THREAD

**Issue by ebouchut** [alexreisner/geocoder#1066]
Hi,

First off, thank you for the gem, it is very useful.

During `rake assets:precompile` I have the following error with either v1.3.5 and v1.3.6.

```
rake aborted!
NoMethodError: private method `include' called for ActionDispatch::Request:Class
.../gems/ruby-2.0.0-p195/gems/geocoder-1.3.5/lib/geocoder/request.rb:82:in `<top (required)>'-
.../gems/ruby-2.0.0-p195/gems/activesupport-4.0.9/lib/active_support/dependencies.rb:229:in `require'
.../gems/ruby-2.0.0-p195/gems/activesupport-4.0.9/lib/active_support/dependencies.rb:229:in `block in require'
.../gems/ruby-2.0.0-p195/gems/activesupport-4.0.9/lib/active_support/dependencies.rb:214:in `load_dependency'
.../gems/ruby-2.0.0-p195/gems/activesupport-4.0.9/lib/active_support/dependencies.rb:229:in `require'
.../gems/ruby-2.0.0-p195/gems/geocoder-1.3.5/lib/geocoder.rb:8:in `<top (required)>'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:72:in `require'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:72:in `block (2 levels) in require'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:70:in `each'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:70:in `block in require'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:59:in `each'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler/runtime.rb:59:in `require'
.../gems/ruby-2.0.0-p195@global/gems/bundler-1.3.5/lib/bundler.rb:132:in `require'
.../config/application.rb:8:in `<top (required)>'
.../Rakefile:4:in `require'
.../Rakefile:4:in `<top (required)>'
(See full trace by running 
```
My understanding is that `Geocoder::Request` cannot call the private method `Module.include`.
> ActionDispatch::Request.include Geocoder::Request

I found a [fix](https://github.com/Teacups/geocoder/blob/bug_no_method_ActionDispatchRequest.include/lib/geocoder/request.rb#L82)
> ActionDispatch::Request.\_\_send\_\_(:include, Geocoder::Request)

I have a [fixed branch](https://github.com/Teacups/geocoder/tree/bug_no_method_ActionDispatchRequest.include) handy, should you be willing to accept a pull-request.
Just let me know what is your take on this issue.

### Environment

- Ruby 2.0.0 (2013-05-14 patchlevel 195) [x86_64-linux] 😞
- Rails: 4.0.9  😞
- Bundler 1.3.5
- Rubygems 2.4.6
- rvm 1.21.14 (stable)

Thanks

**Comment by alepore** [comment]
@ebouchut  TIL `.include` is private on some old ruby versions, sorry for that.
please make the pull request, and please also add the same send call on the other include two line below :)

**Comment by ebouchut** [comment]
Do you want to me rebase the fix branch against master beforehand?

**Comment by ebouchut** [comment]
I rebased, forced pushed and created  PR #1067 .

**Comment by ebouchut** [comment]
Thank you for your help and time.

**Comment by alexreisner** [comment]
Thank YOU for the fix! Releasing 1.3.7 momentarily...

**Comment by ebouchut** [comment]
Great, I confirm it works with Ruby 2.0.0!
